.travis.yml

```diff
@@ -2,9 +2,9 @@ sudo: false
 language: node_js
 
 node_js:
-    - 6
     - 8
-    - node
+    - 10
+    - 12
```

Comment:
We might as well add 14 too, since it is in LTS

Reply:
Thanks. In a future change I'd like to migrate to build over to github actions, We've experienced similar issues in flow-typed where builds take forever because of travis limits